### ISSUE TITLE
JAWS doesn't read aria-labelledby in Firefox with CSS content property

### ISSUE THREAD

**Issue by linda-t** [FreedomScientific/VFO-standards-support#619]
### Summary

> When you code a button with an image using CSS content property, the label that is provided using aria-labelledby is not read in Firefox.  It reads fine in Chrome.
> 
> Example:
1. Create a test-sample.html using below code
2. Create 2 icons, one called twitter.png and the other called twitter2.png
3. Run the test using JAWS in Firefox 

<!DOCTYPE html>
<html>
<head>
<title>Content Image Property</title>
<style>
.icon-twitter {
  content: url(twitter.png);
  display: inline-block;
}

</style>
</head>
<body>
<h1>This test shows aria-labelledby is not honored in Firefox</h1>

<div id="footerSocial">
  <a href="http://www.twitter.com/" class="icon-twitter" role="button" aria-labelledby="test">
  <img id="test" src="twitter2.png" alt="Twitter">
  </a>
</div>

</body>
</html>	

### Expected result

> The button should say Twitter button

### Actual result

> Unlabelled zero button

### Example

> A test case that demonstrates the issue - provide linked test case, CodePen, JSbin, etc...

#### JAWS version and build number
JAWS 2022.2204.20 ILM 

 
#### Operating System and version
Windows 10

#### Browser and version:
Firefox 91.8.0esr(32-bit)

**Comment by JAWS-test** [comment]
It would be good to have the complete source code of your example or an example for testing (e.g. at CodePen).

**Comment by JAWS-test** [comment]
Maybe related: https://github.com/FreedomScientific/VFO-standards-support/issues/297 and https://github.com/FreedomScientific/VFO-standards-support/issues/367

**Comment by linda-t** [comment]
Save this as HTML file and read with JAWS. 
[Fail_aria-labelledby.txt](https://github.com/FreedomScientific/VFO-standards-support/files/8532822/Fail_aria-labelledby.txt)

**Comment by linda-t** [comment]
![twitter](https://user-images.githubusercontent.com/29257603/164510141-91d7d05f-cde3-4fa7-b132-4d2ba7b84112.png)
![twitter2](https://user-images.githubusercontent.com/29257603/164510149-b0c4d767-709e-4295-a741-c20289942dfd.png)

**Comment by JAWS-test** [comment]
To my knowledge, the `content` property is only allowed in conjunction with the pseudo-classes `:before` or `:after`. Therefore your source code seems to be wrong. If `content` would be allowed without `:before` or `:after`, it is a bug of Firefox and not of JAWS, because Firefox submits the button without label to the Accessibility API.
